posts/2021/10/jupyter-community-calls.md

```diff
@@ -0,0 +1,102 @@
+<!--
+.. title: A year of Jupyter community calls
+.. slug: jupyter-community-calls
+.. date: 2021-11-10
+.. author: Isabela Presedo-Floyd, Tony Fast
+.. tags: Jupyter, Community
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+# Jupyter community calls
+
+A framing for open source is that the software and code are kernels of community.
+The code, and its abstractions, unite developers and their patrons; 
+a struggle for growing/evolving open communities is to make sure these groups remain connected. 
+A lot of us showed up for the code, but hung around for the community. 
+We'll continue this post talking about the monthly Jupyter community calls, 
+and how they help all [jovyans](https://jupyter.readthedocs.io/en/latest/community/content-community.html#what-is-a-jovyan), Project Jupyter's pet name for their developers and users, stay connected.
+
+
+## Why community calls?
+
+  > A lot of us showed up for the code, but hung around for the community. 
```

Comment:
Given the discussion above, I assume putting this quote here is intentional to highlight the previous lines?

Reply:
yea it was suggested that we highlight this sentiment, if it is too much then we can remove it

Reply:
It looks really good when rendered! So I agree in this highlight 🌟